mtools/test/test_mlaunch_commandlines.py

```diff
@@ -49,15 +49,15 @@ def read_config(self):
     def cmdlist_filter(self, cmdlist):
         """Filter command lines to contain only [mongod|mongos] --parameter."""
         res = map(lambda cmd: set([param for param in cmd.split()
-                                   if param.startswith('mongo') or
+                                   if param.startswith('"mongo') or
```

Comment:
Can you add a comment about the `"`? Otherwise may be mistaken for a typo.

Feels a tad hacky, but quoting the binary name seems to work cross-platform.